front/src/reducers/user/index.ts

```diff
@@ -7,7 +7,8 @@ export interface UserState {
   isLogged: boolean;
   loginError?: ApiError;
   username: string;
-  userPreferences: { safeWord: string };
+  // TODO PACEDTRAIN: Remove pacedTrain after development pacedTrain feature
+  userPreferences: { safeWord: string; pacedTrain?: boolean };
```

Comment:
`allowPacedTrains` or `showPacedTrains` may be a better name. I think it would be nice to have an explicit name since we are going to introduce a lot of change in the coming weeks

```suggestion
  userPreferences: { safeWord: string; allowPacedTrains?: boolean };
```

Reply:
Done